demo/package.json

```diff
@@ -24,16 +29,22 @@
     "@angular/platform-browser-dynamic": "^7.0.3",
     "@angular/router": "~7.0.3",
     "@angular/upgrade": "~7.0.3",
-    "@hmcts/ccd-case-ui-toolkit": "2.9.0",
-    "@hmcts/ccpay-web-component": "^1.6.2",
+    "@hmcts/ccd-case-ui-toolkit": "2.73.0",
```

Comment:
though this is from demo, is the version need to be updated to latest?

Reply:
@rayliangatos - I'm not sure on this one... @Mo-Lala84?

Reply:
this is a nice to have, the version should be synced with latest